spec/griddler/sendgrid/adapter_spec.rb

```diff
@@ -44,6 +44,33 @@
     normalized_params.should_not have_key(:attachment_info)
   end
 
+  it 'uses sendgrid attachment info for filename' do
```

Comment:
@iwz That's what I thought but I kept quotes in the same style as the rest of the code. Double quotes were needed in the JSON due to parsing errors.

Single quotes to match this repo's existing code or double quotes to match the style guide?